### PR TITLE
Crit LED fixes

### PR DESCRIPTION
**Pull request by philam2001**
### Changelist 

- Enable TV light when TV is being used 
- Enable Regen only when Regen is used and available (not faulted)

### CODE REVIEW

File: firmware/quadruna/VC/src/app/states/app_driveState.c [hunk 1]
```diff
@@ -168,6 +174,9 @@ static void driveStateRunOnExit(void)
     // Disable buzzer on exit drive.
     io_efuse_setChannel(EFUSE_CHANNEL_BUZZER, false);
     app_canTx_VC_BuzzerOn_set(false);
+
+    app_canTx_VC_RegenEnabled_set(false);
+    app_canTx_VC_TorqueVectoringEnabled_set(false);
```
Comment (gtaharaedmonds) [inline firmware/quadruna/VC/src/app/states/app_driveState.c:179]: I believe `VC_TorqueVectoringEnabled` is also not populated anywhere?

**Review by gtaharaedmonds — CHANGES_REQUESTED** [review]
Sort of unrelated but can you consolidated `RegenEnabled` and `TorqueVectoringEnabled` into the same CAN message? Currently there each 1 bit signals with their own message, which adds unnecessary bus load. Better would be to put them in the same message.

File: firmware/quadruna/VC/src/app/states/app_driveState.c (hunk 1 above)
Comment (philam2001) [reply firmware/quadruna/VC/src/app/states/app_driveState.c:179]: It's in the TV init function

**Review by gtaharaedmonds — APPROVED** [review]
LGTM

**Review by Lucien950 — APPROVED** [review]

**Pull request merged**